databricks/koalas/frame.py

```diff
@@ -11097,9 +11098,11 @@ def __setattr__(self, key: str, value) -> None:
         if (key,) in self._internal.column_labels:
             self[key] = value
         else:
-            warnings.warn(
-                "Koalas doesn't allow columns to be created via a new attribute name", UserWarning
-            )
+            msg = "Koalas doesn't allow columns to be created via a new attribute name"
```

Comment:
Quick question. What about we just throw an exception as is for what we don't allow?